testing/tests/functions.rs

```diff
@@ -0,0 +1,46 @@
+use askama::Template;
+
+#[derive(Template)]
+#[template(source = "{{ self::simple_test_fn() }}", ext = "txt")]
+struct FnTemplate;
+
+fn simple_test_fn() -> &'static str {
+    "foo"
+}
+
+#[test]
+fn test_fn() {
+    let t = FnTemplate;
+    assert_eq!(t.render().unwrap(), "foo");
+}
+
+
+#[derive(Template)]
+#[template(source = "{{ self::arg_test_fn(arg) }}", ext = "txt")]
```

Comment:
Don't think we need this test, since we're reusing the same argument parsing that's used for other kinds of calls.